mentors/java/p.json

```diff
@@ -38,5 +38,13 @@
     "link_url": null,
     "avatar_url": "https://i.imgur.com/B43uP3v.jpg",
     "bio": "I am huge fan of java! I really love web, open source software and idea of 'knowledfe for everyone'. At the moment I'm full stack developer, but my passin is Java!"
+  },
+  {
+    "github_username": "leeyspaul",
```

Comment:
The first letter of your username is `l`.  You should be updating `l.json` and not `p.json`.

Reply:
thanks for the catch! Just updated.